frontend/src/api.js

```diff
@@ -1,9 +1,10 @@
 const envVars = window.gitbasepg || { SERVER_URL: '', SELECT_LIMIT: 100 };
 
-const serverUrl = envVars.SERVER_URL || window.location.origin;
+const serverUrl = envVars.SERVER_URL;
 const selectLimit = envVars.SELECT_LIMIT;
 
-const apiUrl = url => `${serverUrl}${url}`;
+// ensure to use relative urls in order to work behind proxies
```

Comment:
This part is tricky, so I would expand more this comment. Something like:
```
If serverUrl is unset, this replaces the leading / in order to work behind proxies with a path other than /.
```

Reply:
Let's apply this comment (or a better one if you see a way to explain it more clearly)

Reply:
Oh yes sorry, complete forgot about that!